usaspending_api/common/mixins.py

```diff
@@ -185,8 +185,8 @@ def build_response(self, request, *args, **kwargs):
         queryset = kwargs.get('queryset')
 
         # workaround to handle both GET and POST requests
-        params = dict(self.request.query_params)
-        params.update(dict(self.request.data))
+        params = self.request.query_params.copy()  # copy() creates mutable copy of a QueryDict
```

Comment:
:thinking: Thanks for the heads-up...this is behaving as expected (and the QueryDict object doesn't have a deepcopy method), so will leave this for now if that's okay by you.